hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/ipc/CallerContext.java

```diff
@@ -113,8 +113,9 @@ public String toString() {
 
   /** The caller context builder. */
   public static final class Builder {
+    private static final String colon = ":";
```

Comment:
The name should be capitalized.
I would also call it something like FIELD_SEPARATOR or KEY_VALUE_SEPARATOR_CHAR.